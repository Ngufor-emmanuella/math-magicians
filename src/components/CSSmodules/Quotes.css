.intro {
  display: flex;
  margin: 15rem auto 0 auto;
  width: 70%;
  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
  font-weight: bolder;
  font-size: 25px;
  color: rgb(19, 19, 85);
}
