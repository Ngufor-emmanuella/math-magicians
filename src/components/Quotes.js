import React from 'react';
import './CSSmodules/Quotes.css';

const Quotes = () => {
  return (
    <div className="quote">
      <h2 className="intro">Mathematics is not about numbers, equations, computations or algorithms; 
      it is about understanding - William Paul Thurston.
      </h2>
    </div>
  );
};
export default Quotes;