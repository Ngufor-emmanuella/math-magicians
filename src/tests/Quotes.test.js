import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

test('should match the snapshot', () => {
  const quote = renderer.create(<Quotes />);
  const tree = quote.toJSON();
  expect(tree).toMatchSnapshot();
});
