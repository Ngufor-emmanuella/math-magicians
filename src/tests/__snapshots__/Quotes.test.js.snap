// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="quote"
>
  <h2
    className="intro"
  >
    Mathematics is not about numbers, equations, computations or algorithms; it is about understanding - William Paul Thurston.
  </h2>
</div>
`;
